templates/_helpers.tpl

```diff
@@ -181,6 +181,11 @@ Reduce duplication of the st2.*.conf volume details
 {{- end }}
 {{- end -}}
 
+{{- define "stackstorm-ha.st2-entrypoint" -}}
+- /usr/bin/dumb-init
+- '--'
```

Comment:
I wonder about pulling this from values. That way we can change the default when the upstream docker images are adjusted. That way if someone is using an older image that doesn't have `dumb-init`, they can still use this chart.